static/js/place/main_pane.tsx

```diff
@@ -132,7 +132,11 @@ class MainPane extends React.Component<MainPanePropType> {
     return (
       <RawIntlProvider value={intl}>
         {this.showOverview() && (
-          <Overview dcid={this.props.dcid} locale={this.props.locale} />
+          <Overview
+            dcid={this.props.dcid}
+            showRanking={this.props.dcid.startsWith("geoId/")}
```

Comment:
maybe just set this as true to keep original behavior of place page

Reply:
Done!